### ISSUE TITLE
Wiki Checklists

### ISSUE THREAD

**Issue by mariaruth** [worldbank/d4di#299]
Link to the printable version? it's an ugly URL, but better than having people land on a page they have to click through. 

https://dimewiki.worldbank.org/index.php?title=Checklist:_Refine_the_Questionnaire_(Content)&printable=yes

**Comment by mariaruth** [comment]
Also, need to review / update all checklists as part of general review of linked wiki pages

**Comment by bbdaniels** [comment]
URL farrrrrr too ugly, I don't mind having people need to click through as the checklists should display well in the wiki by default, right?

**Comment by kbjarkefur** [comment]
For all checklists we want to link to we can create a wiki page where we say something short and link to the check-iist from there. That will create cleaner links for the book.

**Comment by luizaandrade** [comment]
I think this is fine:
https://dimewiki.worldbank.org/wiki/Checklist:_Refine_the_Questionnaire_(Content)

**Comment by mariaruth** [comment]
Yeah, as @luizaandrade linked to the landing pages already exist - it's just that then you have to click through to get to the actual checklist and my worry was that people might not understand that.

**Comment by kbjarkefur** [comment]
Done in 6429cb0827cdb3cdf2ec0837b28c69a9b19b6889

I removed the `&printable=yes` parameter. It is not difficult to find the `printable version` button in the sidebar, and url looks better. and it is easier to see how this fits into the rest of the wiki if it does not land on the printable page.

This can be closed now.

**Comment by mariaruth** [comment]
Either don't close, or document elsewhere that we want to cleanup/revise checklists in wiki as mentioned above
